src/util/ResourceUtil.ts

```diff
@@ -33,3 +38,14 @@ export function generateContainmentQuads(containerURI: NamedNode, childURIs: str
   return new RepresentationMetadata(containerURI,
     { [LDP.contains]: childURIs.map(DataFactory.namedNode) }).quads();
 }
+
+export async function cloneRepresentation(representation: Representation): Promise<BasicRepresentation> {
```

Comment:
Going to need some jsdoc here explaining what the function does. This includes indicating that the original representation can still be used, and that this function loads the entire stream in memory.